src/test/java/com/qa/shopkick/tests/Authentication/LoginHooks.java

```diff
@@ -10,6 +9,9 @@
 import org.apache.log4j.Logger;
 import org.openqa.selenium.support.PageFactory;
 
+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertTrue;
+
```

Comment:
are we using these anywhere in this class?

+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertTrue;